internal/cli/atlas/deployments/setup.go

```diff
@@ -106,11 +106,6 @@ type SetupOpts struct {
 	atlasSetup    *setup.Opts
 }
 
-func (opts *SetupOpts) initPodmanClient() error {
-	opts.podmanClient = podman.NewClient(log.IsDebugLevel(), log.Writer())
```

Comment:
if you're no longer doing it you need to delete the `podmanClient` field from the `SetupOpts` struct